logging/unit_tests/test_client.py

```diff
@@ -42,10 +42,10 @@ def test_logging_api_wo_gax(self):
         from google.cloud.logging.connection import _LoggingAPI
         from google.cloud.logging import client as MUT
         from google.cloud._testing import _Monkey
-        client = self._makeOne(self.PROJECT, credentials=_Credentials())
-        conn = client.connection = object()
 
         with _Monkey(MUT, _USE_GAX=False):
+            client = self._makeOne(self.PROJECT, credentials=_Credentials())
+            conn = client.connection = object()
```

Comment:
Not that it matters but after this change the only thing you need in the context manager is `client = self._makeOne(...)`